examples/efficient_net.py

```diff
@@ -0,0 +1,298 @@
+"""This script provides an example of using efficient for training."""
+import os
+import math
+import tensorflow as tf
+import numpy as np
+from copy import deepcopy
+from tensorflow import keras
+from tensorflow.keras import backend
+from tensorflow.keras import layers
+from tensorflow.keras.models import Model
+from tensorflow.keras.applications import imagenet_utils
+from tensorflow.keras.applications.imagenet_utils import decode_predictions
+from tensorflow.keras.preprocessing import image
+
+from deepreg.model.backbone import Backbone
+from deepreg.registry import REGISTRY
+from deepreg.train import train
+
+efficientnet_params = {
```

Comment:
are these fixed, maybe using UPPER CASE? for `efficientnet_params`

Reply:
Thanks. Done.